test/syscalls/clock_gettime/clock_gettime.c

```diff
@@ -0,0 +1,17 @@
+#include "../../../common/syscalls/syscalls.h"
+#define assert(x) if(!(x)) { asm(".byte 0; .byte 0; .byte 0; .byte 0"); }
+
+int main() {
+  struct __kernel_timespec s, e;
+
+  int ret = rev_clock_gettime(0, &s);
+  assert(ret == 0);
+
+  for(int i = 0; i < 100; i++);
```

Comment:
At `-O2` I'm sure the compiler can optimize away the loop, and even with `volatile`, it takes an infinitesimal amount of time. I'd normally use `nanosleep()`, but that's another system call which would need to be added.

Reply:
This is just for demonstrative purposes. Even if I throw it away it still will pass because an assert at the end contains =0 case. 
Do you want me to remove it from the test/example or add a comment?

Reply:
Maybe add `volatile` and a comment that this is for illustration purposes.

Reply:
I doubt volatile will prevent compiler from removing dead code, instead I used argc which is not known during compilation time and the worst case is that the loop will be changed to multiplication, which is fine I guess.